modules/context/api.go

```diff
@@ -8,6 +8,7 @@ import (
 	"fmt"
 	"strings"
 
+        "code.gitea.io/gitea/models"
```

Comment:
this line isn't `gofmt`. I'm going to merge anyway, as this is minor and we can update with another PR. Probably there's more like that anyway which we can push all together.